code/game/machinery/transformer.dm

```diff
@@ -6,25 +6,51 @@
 	layer = MOB_LAYER+1 // Overhead
 	anchored = 1
 	density = 1
-	var/transform_dead = 0
-	var/transform_standing = 0
-	var/cooldown_duration = 600 // 1 minute
-	var/cooldown = 0
-	var/robot_cell_charge = 5000
+	/// TRUE if the factory can transform dead mobs.
+	var/transform_dead = TRUE
+	/// TRUE if the mob can be standing and still be transformed.
+	var/transform_standing = TRUE
+	/// Cooldown between each transformation, in deciseconds.
+	var/cooldown_duration = 600
```

Comment:
can use `1 MINUTE` here

Reply:
Yep.

Reply:
I forgot you can't do this in variable declarations:

![Code_M0mr6TDlAh](https://user-images.githubusercontent.com/42044220/87950622-2a692000-ca6d-11ea-8bae-429cdfb8f52b.png)

And the compiler line says: `code\game\machinery\transformer.dm:14:error: MINUTE: expected end of statement`

Reply:
Oh yeah it's `MINUTES` instead of MINUTE. Should compile with that

Reply:
Oh... right, it's plural. Works now :smile: